Dockerfile.in

```diff
@@ -1,29 +1,41 @@
 FROM DISTRO
+
 # Docker build arguments
 ARG SOURCE_DIR=/ffmpeg
 ARG ARTIFACT_DIR=/dist
+
 # Docker run environment
-ENV DEB_BUILD_OPTIONS=noddebs
-ENV DEBIAN_FRONTEND=noninteractive
-ENV ARCH=BUILD_ARCHITECTURE
-ENV GCC_VER=GCC_RELEASE_VERSION
-ENV LLVM_VER=LLVM_RELEASE_VERSION
-ENV SOURCE_DIR=/ffmpeg
-ENV ARTIFACT_DIR=/dist
-ENV TARGET_DIR=/usr/lib/jellyfin-ffmpeg
-ENV DPKG_INSTALL_LIST=${SOURCE_DIR}/debian/jellyfin-ffmpeg6.install
-ENV PATH=${TARGET_DIR}/bin:${PATH}
-ENV PKG_CONFIG_PATH=${TARGET_DIR}/lib/pkgconfig:${PKG_CONFIG_PATH}
-ENV LD_LIBRARY_PATH=${TARGET_DIR}/lib:${TARGET_DIR}/lib/mfx:${TARGET_DIR}/lib/xorg:${LD_LIBRARY_PATH}
-ENV LDFLAGS="-Wl,-rpath=${TARGET_DIR}/lib -L${TARGET_DIR}/lib"
-ENV CXXFLAGS="-I${TARGET_DIR}/include $CXXFLAGS"
-ENV CPPFLAGS="-I${TARGET_DIR}/include $CPPFLAGS"
-ENV CFLAGS="-I${TARGET_DIR}/include $CFLAGS"
+ENV DEBIAN_FRONTEND=noninteractive \
```

Comment:
It seems you can't use the first var in the second var when setting multiple env var in single line. https://stackoverflow.com/a/45529293

Reply:
So I should revert this and keep env for every single line ?

Reply:
With this changed the script can build ?

Reply:
Didn't need to change this as it works with the change I made, don't need to separate in two of them

Reply:
@Gauvino Please refer to the `FFBUILD_TOOLCHAIN` in this file. It's a copy of https://github.com/BtbN/FFmpeg-Builds, they are more expert on this than me.

https://github.com/jellyfin/jellyfin-ffmpeg/blob/bc59f46164aa687e2d571b31b493b87cc1a611a5/builder/images/base-linux64/Dockerfile#L42-L61